src/utils/events/move-input.js

```diff
@@ -24,9 +26,29 @@ export default class MoveInput {
   }
 
   handler(event) {
```

Comment:
No reason you should have known. I'm ambivalent about it, actually -- that's just the origin of the name, not an argument for or against it. I agree that it's not a great name so in a future pass through here perhaps we can change it.